conda/python/meta.yaml

```diff
@@ -0,0 +1,59 @@
+# Making a Change to this package?
+# REMEMBER TO UPDATE the .yaml files for the following packages:
+#   moose/conda_build_config.yaml
+# As well as any directions pertaining to modifying those files.
+{% set version = "2025.10.02" %}
+
+package:
+  name: moose-python
```

Comment:
I think this might conflict with the internal name I already used in `moose-python` with the MPI package: https://github.com/idaholab/moose/blob/ee0740de41d6d9f880bc7bc261a6d100a8c92a40/conda/mpi/meta.yaml#L109